src/components/MultipleAvatars.js

```diff
@@ -265,7 +300,7 @@ function MultipleAvatars(props) {
                                         <Avatar
                                             source={props.icons[1].source || props.fallbackIcon}
                                             fill={themeColors.iconSuccessFill}
-                                            size={props.isFocusMode ? CONST.AVATAR_SIZE.MID_SUBSCRIPT : CONST.AVATAR_SIZE.SMALLER}
+                                            size={avatarSize}
```

Comment:
This affects all avatars throughout the app, have we tested for regressions on avatars?